collector/mssql.go

```diff
@@ -137,49 +137,50 @@ type MSSQLCollector struct {
-	AccessMethodsUsedtreepagecookie           *prometheus.Desc
-	AccessMethodsWorkfilesCreated             *prometheus.Desc
-	AccessMethodsWorktablesCreated            *prometheus.Desc
-	AccessMethodsWorktablesFromCacheRatio     *prometheus.Desc
+	AccessMethodsAUcleanupbatches              *prometheus.Desc
+	AccessMethodsAUcleanups                    *prometheus.Desc
+	AccessMethodsByreferenceLobCreateCount     *prometheus.Desc
+	AccessMethodsByreferenceLobUseCount        *prometheus.Desc
+	AccessMethodsCountLobReadahead             *prometheus.Desc
+	AccessMethodsCountPullInRow                *prometheus.Desc
+	AccessMethodsCountPushOffRow               *prometheus.Desc
+	AccessMethodsDeferreddroppedAUs            *prometheus.Desc
+	AccessMethodsDeferredDroppedrowsets        *prometheus.Desc
+	AccessMethodsDroppedrowsetcleanups         *prometheus.Desc
+	AccessMethodsDroppedrowsetsskipped         *prometheus.Desc
+	AccessMethodsExtentDeallocations           *prometheus.Desc
+	AccessMethodsExtentsAllocated              *prometheus.Desc
+	AccessMethodsFailedAUcleanupbatches        *prometheus.Desc
+	AccessMethodsFailedleafpagecookie          *prometheus.Desc
+	AccessMethodsFailedtreepagecookie          *prometheus.Desc
+	AccessMethodsForwardedRecords              *prometheus.Desc
+	AccessMethodsFreeSpacePageFetches          *prometheus.Desc
+	AccessMethodsFreeSpaceScans                *prometheus.Desc
+	AccessMethodsFullScans                     *prometheus.Desc
+	AccessMethodsIndexSearches                 *prometheus.Desc
+	AccessMethodsInSysXactwaits                *prometheus.Desc
+	AccessMethodsLobHandleCreateCount          *prometheus.Desc
+	AccessMethodsLobHandleDestroyCount         *prometheus.Desc
+	AccessMethodsLobSSProviderCreateCount      *prometheus.Desc
+	AccessMethodsLobSSProviderDestroyCount     *prometheus.Desc
+	AccessMethodsLobSSProviderTruncationCount  *prometheus.Desc
+	AccessMethodsMixedpageallocations          *prometheus.Desc
+	AccessMethodsPagecompressionattempts       *prometheus.Desc
+	AccessMethodsPageDeallocations             *prometheus.Desc
+	AccessMethodsPagesAllocated                *prometheus.Desc
+	AccessMethodsPagescompressed               *prometheus.Desc
+	AccessMethodsPageSplits                    *prometheus.Desc
+	AccessMethodsProbeScans                    *prometheus.Desc
+	AccessMethodsRangeScans                    *prometheus.Desc
+	AccessMethodsScanPointRevalidations        *prometheus.Desc
+	AccessMethodsSkippedGhostedRecords         *prometheus.Desc
+	AccessMethodsTableLockEscalations          *prometheus.Desc
+	AccessMethodsUsedleafpagecookie            *prometheus.Desc
+	AccessMethodsUsedtreepagecookie            *prometheus.Desc
+	AccessMethodsWorkfilesCreated              *prometheus.Desc
+	AccessMethodsWorktablesCreated             *prometheus.Desc
+	AccessMethodsWorktablesFromCacheRatio      *prometheus.Desc
+	AccessMethodsWorktablesFromCacheRatio_Base *prometheus.Desc
```

Comment:
It would be nice to not leak the counter-internal names into our "clean" structs, but rather have the names here reflect what they mean. So something like `AccessMethodsWorktablesFromCacheLookups`?
(This goes for all the new fields added below as well)